src/app/public/modules/select-field/select-field.component.ts

```diff
@@ -100,14 +109,19 @@ export class SkySelectFieldComponent implements ControlValueAccessor {
   @Input()
   public pickerHeading: string;
 
+  @Output()
+  public blur = new EventEmitter();
+
   public get value(): any {
     return this._value;
   }
 
   public set value(value: any) {
-    this._value = value;
-    this.onChange(this.value);
-    this.onTouched();
+    if (JSON.stringify(this._value) !== JSON.stringify(value)) {
```

Comment:
We're not making this check in the `master` branch. Does your new "blur" feature need it?

Reply:
So I wanted to reduce the number of touch and inputChanged updates from value changes to ignore when a value hasn't actually been changed. It was a nicety that would also let me remove the safeguards in our service where we have to filter value updates to be unique.

It's not really necessary for the blur event, but something that looked like a good idea overall and fixed another problem I saw bandaids for

Reply:
Oh yeah, for sure. Sorry, I had convinced myself that I had already written a test for this :/
But it isn't there. I'll add them